src/node.cc

```diff
@@ -1546,11 +1557,28 @@ static void GetActiveRequests(const FunctionCallbackInfo<Value>& args) {
   Environment* env = Environment::GetCurrent(args);
 
   Local<Array> ary = Array::New(args.GetIsolate());
-  int i = 0;
+  Local<Context> ctx = env->context();
+  Local<Function> fn = env->add_properties_by_index_function();
+  Local<Value> argv[5];
+  size_t i = 0;
+
+  for (auto w : *env->req_wrap_queue()) {
+    if (w->persistent().IsEmpty() == false) {
+      argv[i++ % 5] = w->object();
```

Comment:
Can you use `ARRAY_SIZE(argv)` instead of hard-coding it in several places?  If it gets unwieldy, I suggest writing it as:

```
static const size_t argc = 5;
Local<Value> argv[argc];
// ...
argv[i++ % argc] = ...;
```